src/controller/python/build-chip-wheel.py

```diff
@@ -0,0 +1,210 @@
+    # directory is writable.
+    chipPackageDir = os.path.join(tmpDir, packageName)
+    if os.path.isdir(chipPackageDir):
+        shutil.rmtree(chipPackageDir)
+    shutil.copytree(os.path.join(buildDir, 'chip'), chipPackageDir)
+    os.chmod(chipPackageDir, os.stat(chipPackageDir).st_mode|stat.S_IWUSR)
+    
+    # Copy the chip wrapper DLL from where libtool places it (.libs) into
+    # the root of the chip package directory.  This is necessary because
+    # setuptools will only add package data files that are relative to the
+    # associated package source root.
+    shutil.copy2(os.path.join(buildDir, 'chip', chipDLLName),
+                 os.path.join(chipPackageDir, chipDLLName))
+    
+    # Make a copy of the Chip Device Controller Shell script in the tmp directory,
+    # but without the .py suffix. This is how we want it to appear when installed
+    # by the wheel.
+    shutil.copy2(os.path.join(buildDir, deviceManagerShellName),
+                 os.path.join(tmpDir, chipControllerShellInstalledName))
+    
+    # Search for the CHIP LICENSE file in the parents of the source
+    # directory and make a copy of the file called LICENSE.txt in the tmp
+    # directory.  
+    def _AllDirsToRoot(dir):
+        dir = os.path.abspath(dir)
+        while True:
+            yield dir
+            parent = os.path.dirname(dir)
+            if parent == '' or parent == dir:
+                break
+            dir = parent
+    for dir in _AllDirsToRoot(buildDir):
+        licFileName = os.path.join(dir, 'LICENSE')
+        if os.path.isfile(licFileName):
+            shutil.copy2(licFileName,
+                         os.path.join(tmpDir, 'LICENSE.txt'))
+            break
+    else:
+        raise FileNotFoundError('Unable to find CHIP LICENSE file')
+    
+    # Define a custom version of the bdist_wheel command that configures the
+    # resultant wheel as platform-specific (i.e. not "pure"). 
+    class bdist_wheel_override(bdist_wheel):
+        def finalize_options(self):
+            bdist_wheel.finalize_options(self)
+            self.root_is_pure = False
+    
+    # Construct the package version string.  If building under CHIP use the CHIP
+    # build number as the package version.  Otherwise use a dummy version of '0.0'.
+    # (See PEP-440 for the syntax rules for python package versions).
+    if 'CHIP_BUILD_NUMBER' in os.environ:
+        chipPackageVer = os.environ['CHIP_BUILD_NUMBER']
+    else:
+        chipPackageVer = os.environ.get('CHIP_PYTHON_VERSION', '0.0')
+    
+    # Generate a description string with information on how/when the package
+    # was built. 
+    if 'CHIP_BUILD_NUMBER' in os.environ:
+        buildDescription = 'Built by CHIP CI on %s\n- Build: %s/#%s\n- Build URL: %s\n- Branch: %s\n- Commit: %s\n' % (
+                                datetime.now().strftime('%Y/%m/%d %H:%M:%S'),
+                                os.environ['CHIP_REPO_SLUG'],
```

Comment:
Please pass all this on the command line.

Reply:
just remove the extra environment, at this moment, we don't need it for wheel.